### DISCUSSION TITLE
Feature request: add non-Soundcloud audio embed as possible media src

### DISCUSSION THREAD

**Discussion by grantbarrett** [hyperaudio/wordpress-hyperaudio#119] [General] [answered]
I'm pleased at the features offered by Hyperaudio but I am currently moving away from SoundCloud embeds toward WordPress audio block embeds that link MP3 audio files in a podcast feed. I'd love it if Hyperaudio would work with standard audio file formats, too. Thanks for considering this, if it's even possible.

**Comment by maboa** [comment]
Will definitely look into it.

**Comment by maboa** [comment]
Checked and this is already possible:

Something like:

```
[hyperaudio src="http://example.com/drbob.mp3"]<article><section><p><span data-m="1290" data-d="490">This, </span><span data-m="2100" data-d="270">co-</span><span data-m="2370" data-d="629">pilot </span><span data-m="3150" data-d="359">"Dr </span><span data-m="3510" data-d="410">Bob" </span><span data-m="4200" data-d="679">rocks!</span></p></section></article>[/hyperaudio]
```

Was there another audio format you wanted to support @gumbolastima ?

**Comment by maboa** [comment, marked as answer]
You can also achieve compatibility with other native audio formats by using the `player` attribute. ie:

```
[hyperaudio src="http://example.com/drbob.ogg" player="nativeAudio"]<article><section><p><span data-m="1290" data-d="490">This, </span><span data-m="2100" data-d="270">co-</span><span data-m="2370" data-d="629">pilot </span><span data-m="3150" data-d="359">"Dr </span><span data-m="3510" data-d="410">Bob" </span><span data-m="4200" data-d="679">rocks!</span></p></section></article>[/hyperaudio]
```

**Comment by ataha24** [comment]
Following this issue...really happy with wp plugin and beyond excited to integrate it for a podcast! I was wondering about incorperating media from other sources (i.e., anchorfm or google podcasts)? Limited expertise in this domain, but I think these dont point to ".mp3" files directly? Alternatively, what are some recommended ways to link .mp3 files (too large to host on wp domain) bypassing soundcloud upload (dont have the pro version for unlimited upload size)?

**Reply by maboa** [reply]
Yup. Great idea! 👍🏼  We should build the compatibility into Hyperaudio Lite, since the plugin uses that. See this related issue https://github.com/hyperaudio/hyperaudio-lite/issues/156

**Comment by maboa** [comment]
@gumbolastima @ataha24 the latest version now has Spotify podcast support, [albeit a little limited as the API isn't the best](https://github.com/hyperaudio/hyperaudio-lite/pull/178#issue-1996727339). I also tried adding SimpleCast but there is no API, so I'm not sure it's possible for security reasons, however you can just plug a simplecast source URL into the native player – so another possibility for hosting your podcasts.
